wp-youtube-lyte.php

```diff
@@ -81,15 +84,17 @@
 }
 
 /** get other options and push in array*/
-$lyteSettings['sizeArray']  = $sArray;
-$lyteSettings['selSize']    = $selSize;
-$lyteSettings['links']      = get_option('lyte_show_links');
-$lyteSettings['file']       = $wyl_file."?wyl_version=".$wyl_version;
-$lyteSettings['ratioClass'] = ( $pSizeFormat === '43' ) ? ' fourthree' : '';
-$lyteSettings['pos']        = ( get_option( 'lyte_position', '0' ) === '1' ) ? 'margin:5px auto;' : 'margin:5px;';
-$lyteSettings['microdata']  = get_option( 'lyte_microdata', '1' );
-$lyteSettings['hidef']      = get_option( 'lyte_hidef', 0 );
-$lyteSettings['scheme']     = ( is_ssl() ) ? "https" : "http";
+$lyteSettings['sizeArray']                    = $sArray;
+$lyteSettings['selSize']                      = $selSize;
+$lyteSettings['links']                        = get_option('lyte_show_links');
+$lyteSettings['file']                         = $wyl_file."?wyl_version=".$wyl_version;
+$lyteSettings['file_lazyload']                = $wyl_file_lazyload."?wyl_version=".$wyl_version;
+$lyteSettings['lyte_use_internal_lazyloader'] = apply_filters('lyte_use_internal_lazyloader', false );
```

Comment:
Afraid that at the point where you added the filter now, it triggers much to early (as the setting of lyteSettings is done immediately, not hooking into any wordpress core action). Best would be to add it in the lyte_settings_enforcer function right before lyte_settings filter is called which hooks into after_setup_theme?

Reply:
Good point. I checked to make sure we're not trying to use $lyteSettings['lyte_use_internal_lazyloader'] before the lyte_settings_enforcer function gets called and it looks fine.